backend/sites/src/app/middleware/userJwtTokenDecoder.ts

```diff
@@ -22,6 +22,7 @@ export class UserJWTTokenDecoderMiddleware implements NestMiddleware {
         const { email, sub, identity_provider, given_name, family_name } =
           decodedToken;
 
+        req['user'] = decodedToken;
```

Comment:
what this code will do here ?

Reply:
This sets decoded token info on the request object. It allows accessing [the current user at the time of the request](https://github.com/bcgov/nr-site-registry/pull/63/files#diff-48df5eb045a176ba7be68e1b7c753927d63656242a0d6cf0e3105bd8e46c2ea2R64)

Reply:
ok